jinahub/indexers/SimpleIndexer/simple_indexer.py

```diff
@@ -93,10 +94,11 @@ def search(
         top_k = int(parameters.get('top_k', self.default_top_k))
         flat_docs.match(
             self._docs,
-            metric=lambda q_emb, d_emb, _: self.distance(
+            metric=lambda q_emb, d_emb: self.distance(
                 _ext_A(_norm(q_emb)), _ext_B(_norm(d_emb))
```

Comment:
It looks like an error. The `_norm` should not be applied for the `euclidean` metric.

Reply:
I agree.